README.md

```diff
@@ -263,6 +263,24 @@ engine_name = 'favorite-videos'
 client.reset_settings(engine_name)
 ```
 
+#### Create a Signed Search Key
+
+Creating a search key that will only return the title field.
+
+```ruby
+read_only_api_key = 'search-xxxxxxxxxxxxxxxxxxxxxxxx'
```

Comment:
Same comment as in app-search-node:

That is minor, but I would use the same words that are used in the app:
publicSearchKey or publicSearchApiKey
![image](https://user-images.githubusercontent.com/11838280/63541710-0d426f00-c4f5-11e9-8698-4c5f211bdfe7.png)
The current naming may lead to confusion because we have read/write mode for a private key:
<img width="963" alt="Screen Shot 2019-08-22 at 3 54 21 PM" src="https://user-images.githubusercontent.com/11838280/63541762-2cd99780-c4f5-11e9-88ca-5effb1d3de26.png">